DXRMapFixups/DeusEx/Classes/DXRFixupM08.uc

```diff
@@ -441,34 +441,35 @@ function PostFirstEntryMapFixes()
     {
         case "08_NYC_STREET":
 
-            //Place reinforcement points on the cops
-            foreach AllActors(class'#var(prefix)ScriptedPawn',sp){
-                rpTag = FindReinforcementPointTag(sp.Tag);
-                if (rpTag!=''){
-                    reinforce = sp.Spawn(class'DXRReinforcementPoint',,rpTag,sp.Location);
-                    reinforce.Init(sp);
+            if (class'MenuChoice_BalanceMaps'.static.MinorEnabled()){
+                //Place reinforcement points on the cops
+                foreach AllActors(class'#var(prefix)ScriptedPawn',sp){
+                    rpTag = FindReinforcementPointTag(sp.Tag);
+                    if (rpTag!=''){
+                        reinforce = sp.Spawn(class'DXRReinforcementPoint',,rpTag,sp.Location);
+                        reinforce.Init(sp);
+                    }
                 }
-            }
 
-            //Make sure the troopers are running to the reinforcement points
-            foreach AllActors(class'#var(prefix)ScriptedPawn',sp){
-                SetUNATCOTargetOrders(sp);
-            }
+                //Make sure the troopers are running to the reinforcement points
+                foreach AllActors(class'#var(prefix)ScriptedPawn',sp){
+                    SetUNATCOTargetOrders(sp);
+                }
 
-            //A point for the MJ12 Attack Force to run to (the stairs of Osgoode & Sons)
-            //This location works for both Vanilla maps and Revision
-            reinforce = Spawn(class'DXRReinforcementPoint',,'MJ12AttackForceTarget',vectm(530,1900,-450));
-            reinforce.SetCollisionSize(200,100); //Make it bigger than the other reinforcement points
+                //A point for the MJ12 Attack Force to run to (the stairs of Osgoode & Sons)
+                //This location works for both Vanilla maps and Revision
+                reinforce = Spawn(class'DXRReinforcementPoint',,'MJ12AttackForceTarget',vectm(530,1900,-450));
+                reinforce.SetCollisionSize(200,100); //Make it bigger than the other reinforcement points
 
-            //Make the Attack Force run to the reinforcement point instead of directly attacking the player
-            foreach AllActors(class'#var(prefix)ScriptedPawn',sp){
-                if (sp.Tag!='MJ12AttackForce' && sp.Tag!='MJ12AttackForce_clone') continue;
+                //Make the Attack Force run to the reinforcement point instead of directly attacking the player
+                foreach AllActors(class'#var(prefix)ScriptedPawn',sp){
+                    if (sp.Tag!='MJ12AttackForce' && sp.Tag!='MJ12AttackForce_clone') continue;
 
-                sp.bKeepWeaponDrawn=True;
-                sp.SetOrders('RunningTo','MJ12AttackForceTarget',True);
+                    sp.bKeepWeaponDrawn=True;
+                    sp.SetOrders('RunningTo','MJ12AttackForceTarget',True);
+                }
             }
```

Comment:
maybe the cops->unatco reinforcements should be minor fixes, and the raid should be moderate? or all at the same level?

Reply:
I think the "Run to a specific point instead of the player" is one general fix (for both the reinforcements and the attack force), probably both moderate, since it is explicitly different from vanilla.